lib/theme/hs_text_theme.dart

```diff
@@ -28,93 +28,114 @@ class FontStyleGuide {
   static const FontWeight fwRegular = FontWeight.w400;
   static const FontWeight fwSemibold = FontWeight.w600;
   static const FontWeight fwBold = FontWeight.w700;
-
-  static const double height = 1.5;
 }
 
 const TextTheme textTheme = TextTheme(
   //TITLE
   titleLarge: TextStyle(
-      fontSize: FontStyleGuide.fontSize34,
-      fontWeight: FontStyleGuide.fwRegular,
-      letterSpacing: FontStyleGuide.letterSpacing025,
-      height: FontStyleGuide.height),
+    fontSize: FontStyleGuide.fontSize34,
```

Comment:
I just checked and we already have the Manrope font in assets. But would it not be better if I just set the fontFamily in the hs_theme file instead of here, since it would apply the font globally?